test/repository/SchedulesRepository.integration.spec.ts

```diff
@@ -62,7 +64,7 @@ describe('SchedulesRepository', () => {
       const schedules = await Promise.all(
```

Comment:
Rename variable to `schedulesActiveStatus` or something. For a second I thought our repository was broken when I saw that `schedules` is just an array of booleans. :D